x-pack/plugins/stack_connectors/server/connector_types/jira/mocks.ts

```diff
@@ -120,6 +120,9 @@ const executorParams: ExecutorSubActionPushParams = {
     priority: 'High',
     issueType: '10006',
     parent: null,
+    customFields: {
```

Comment:
What about leaving the mock params as it is and creating dedicated tests for custom fields that test all possible scenarios with mocks constructed specially for these tests?

Reply:
Because of our conversation in a previous PR(I think custom fields for cases) I also thought about that :D 

But here, this mock always needs to have `customFields` declared because of the `Incident` type.

And since I had to make the change anyway I put data.

Alternatively, I can have `customFields: null` and change some tests. wdyt?

Reply:
Whatever works best for you. I think one or two dedicated tests for custom fields would be useful even if they are implicitly being tested somewhere else.